bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateGuideStarsTableModel.java

```diff
@@ -165,12 +170,12 @@ public String getId() {
      * Returns a list of SkyObjects corresponding to the checked (or unchecked) rows in the table
      * @param checked if true, return the checked rows (candidates), otherwise the unchecked (non-candidates)
```

Comment:
I suppose we should remove the comment there since `checked` is no longer a param.  Why did we need that before?